data/webbrowser.yaml

```diff
@@ -833,14 +833,14 @@ sources:
 - type: FILE
   attributes:
     paths:
-    - '%%users.localappdata%%\Chromium\User Data\*\Session Storage\Session_*'
```

Comment:
are these paths incorrect or from an older version? should the definition contain both?

Reply:
Initially commit was incorrect.  Fixed now.